.github/workflows/build-images.yml

```diff
@@ -31,13 +31,13 @@ jobs:
           install: true
       - name: Login to DockerHub
         uses: docker/login-action@v3
-        if: github.event_name != 'pull_request' || github.event_name == 'schedule'
+        if: ${{ github.event_name != 'pull_request' && (github.event_name == 'schedule' || startsWith(github.ref, 'refs/tags/')) }}
```

Comment:
What is the purpose/function of wrapping the expression into `${{ }}` brackets?

Reply:
Required syntax/best practise see https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution#overview